### ISSUE TITLE
Current 3.9.0 release is missing minified js files!

### ISSUE THREAD

**Issue by willfaulds** [woocommerce/storefront#1800]
## Describe the bug
The current 3.9.0 release does not contain minimised i.e. `.min.js` javascript assets

Isolating the problem (mark completed items with an [x]):
- [x] I have confirmed the previous release did contain minified files [Wordpress Repository v3.8.1 ](https://themes.trac.wordpress.org/browser/storefront/3.8.1/assets/js?rev=153415) and that the current does not [Wordpress Repository v3.9.0](https://themes.trac.wordpress.org/browser/storefront/3.9.0/assets/js)

**This needs to be fixed ASAP as it is a site-breaking change that can be pulled through via the AutoUpdater!**

**Comment by dsmithweb** [comment]
4330159-zen

**Comment by tjcafferkey** [comment]
After some debugging, I found this was due to a conflict with node versions when running the build for the new Storefront update. Running node version `12.21.0` built the minified JS files.

**I've since done a `3.9.1` release and can confirm this fixes the issue.** But will keep this issue open until it's tested by someone other than me.

**Testing instructions**
* Update your theme to version `3.9.1` via WP Admin > Themes or by downloading it here https://wordpress.org/themes/storefront/
* Install it to your WordPress site
* Open the console on the frontend and no 404 errors should appear for minified JS files
* Smoke testing the theme should not result in any regressions.
